auth-web/src/components/auth/staff/continuation-in/ExtraprovincialRegistrationBc.vue

```diff
@@ -0,0 +1,126 @@
+<template>
+  <div
+    v-if="!!continuationIn"
```

Comment:
Don't render this component until/unless we have data.

I should be able to remove some of the conditional chaining (`a?.b`) with this.

Reply:
I've decided to keep the conditional chaining in place, so that the computeds don't depend on the template. (Ie, the computeds are only evaluated when referenced in the template, and the template won't render without the main object.)